javascript-if/main.js

```diff
@@ -0,0 +1,122 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+//isUnderFive(number)
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  };
+};
+
+console.log('isUnderFive(4):', isUnderFive(4));
+console.log('isUnderFive(10):', isUnderFive(10));
+console.log('isUnderFive(5):', isUnderFive(5));
+
+
+//isEven(number)
+function isEven(number){
+  let remainder = (number % 2); //why do I have to use 'let' instead of 'var' for this to work here?
+  if (remainder === 0) { //why does this only work when I use strict equality?
```

Comment:
in my testing it worked without strict equality, although you should always use strict equality!